Automattic/wp-calypso
# Jetpack Cloud: PoC v3 (AKA "barebone")

This PR creates a foundation for the Jetpack Cloud. The idea is to be able to build it in Calypso, and later, in another PR, to either prepare the infrastructure to ship it together with Calypso but on a different domain, or to build it as a package or app and distribute a production version of it to another domain. In both cases, the domain is expected to be `cloud.jetpack.com`.

This PR is a simpler alternative to #38770 that removes all the state, reducers and app boot middleware and allows for the iterative building of the Jetpack Cloud and related projects. It provides us with a canvas to be able to build the UI of Jetpack Backup. 

Next steps: In subsequent PRs, we still have to add a more complex boot sequence, with a personalized redux store, any necessary middleware and all the fundamentals we're used to having when we develop with Calypso. In parallel with those framework updates, we can also iterate on the UI itself, build base styles and any needed components to kick-start the UIs of Jetpack Cloud.

#### Changes proposed in this Pull Request

* Jetpack Cloud: Add fundamentals
  * Entry point
  * Basic barebone boot
  * Section definition
  * Layout
  * Masterbar
  * Sidebar
  * Example routes
  * Controller
  * Very basic stylesheet

#### Testing instructions

* Checkout this branch.
* Build Calypso.
* Go to http://calypso.localhost:3000/jetpack-cloud
* Verify you can navigate through the example routes in the sidebar.
